src/Phpbb/TranslationValidator/Validator/FileListValidator.php

```diff
@@ -201,6 +201,11 @@ public function validate()
 					}
 					$this->output->addMessage($level, 'Found additional file', $origin_file);
 				}
+
+				if (substr($origin_file, -13 ) === 'site_logo.gif' || substr($origin_file, -14 ) === '/site_logo.gif')
```

Comment:
Space after number can be removed.
Also, this condition makes no sense at all. If the second condition is true, the first condition is true, too, so the second one will never be reached.